### ISSUE TITLE
amp-sidebar test flaky

### ISSUE THREAD

**Issue by zhouyx** [ampproject/amphtml#3097]
```
Chromium 45.0.2454 (Ubuntu 0.0.0) amp-sidebar should open from right is side right is specified FAILED
	Error: timeout of 2000ms exceeded. Ensure the done() callback is being called in this test.
Chromium 45.0.2454 (Ubuntu 0.0.0): Executed 1882 of 1968 (1 FAILED) (skipped 21) (0 secs / 32.836 secs)
ium 45.0.2454 (Ubuntu 0.0.0): Executed 1883 of 1968 (1 FAILED) (skipped 21) (0 secs / 33.209 secs)
```
https://travis-ci.org/ampproject/amphtml/builds/125726478

**Comment by ericlindley-g** [comment]
Putting in current milestone, to get eyes on the flaky test (if not fix it)

**Comment by mkhatib** [comment]
This should be fixed with #3313 in place. If this happens again please re-open.
